homeassistant/components/litterrobot/sensor.py

```diff
@@ -163,12 +173,19 @@ async def async_setup_entry(
         if isinstance(robot, robot_type)
         for description in entity_descriptions
     )
+    async_add_entities(
```

Comment:
Please collect the entities in a single list and pass that to the `async_add_entities` so its only called once